src/main/java/seedu/intern/logic/parser/ParserUtil.java

```diff
@@ -39,6 +39,25 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
         return Index.fromOneBased(Integer.parseInt(trimmedIndex));
     }
 
+    /**
+     * Parses {@code oneBasedIndex/ ALL keyword} into an {@code Index} and
+     * returns it. Leading and trailing whitespaces will be trimmed.
+     * @throws ParseException if the specified index is invalid (not non-zero unsigned integer).
+     */
+    public static Index parseDelete(String oneBasedIndex) throws ParseException {
```

Comment:
`parseDeleteIndex` might be a better name. That being said, it would make more sense if we could agree on the implementation of multi delete:

1. Make delete all a separate command
2. Modify the original parseIndex, and have both edit all/delete all use that index format.